resources/zib/zib-PulseRate.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-PulseRate" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-PulseRate" />
+  <name value="ZibPulseRate" />
+  <title value="zib PulseRate" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The pulse rate is the number of palpations per minute measured at an artery." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) PulseRate-v3.3(2020EN)](https://zibs.nl/wiki/PulseRate-v3.3(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-pulserate-v3.3-2020EN" />
+    <uri value="https://zibs.nl/wiki/PulseRate-v3.3(2020EN)" />
+    <name value="zib PulseRate-v3.3(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/pattern-VitalSigns" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
```

Comment:
The comment on the root that is derived from the pattern-VitalSigns profile should be changed, as the text is not correct in this case (PulseRate not being an 'official' vital sign based on core FHIR). The comment should at least indicate that we've chosen to base PulseRate on the pattern-VitalSigns profile due to it being closely related to HeartRate (and thus also for consistency), even though it is not included in the core VitalSigns profile.